x-pack/test_serverless/api_integration/test_suites/common/reporting/generate_csv_discover.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SerializedSearchSourceFields } from '@kbn/data-plugin/common';
+import expect from '@kbn/expect';
```

Comment:
For some reason, I'm getting an error in my editor saying this line should be `@kbn/expect/expect`. Looking into this.